packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/index.js

```diff
@@ -29,9 +31,13 @@ const CellContent = ({ content, fieldSchema, metadatas, name, queryInfos, rowId
 
       return <MultipleMedias value={content} />;
 
-    case 'relation':
+    case 'relation': {
+      if (isSingleRelation(fieldSchema.relation)) {
```

Comment:
Can you add a statement to return `-` in case it's a polymorphique relation? such as `fieldSchema.relation.includes('morph') && fieldSchema.relation !== 'oneToOneMorph'` ?

Reply:
I believe this case is already covered in `hasContent()`. I've added more tests in 1da94ccc6b979154d572fab704f4a6e320779a60 . Do you agree or did you mean something else?